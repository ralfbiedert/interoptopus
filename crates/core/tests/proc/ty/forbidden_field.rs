use interoptopus::ffi_type;

#[ffi_type]
struct Foo {
    new: u8,
}

fn main() {}
