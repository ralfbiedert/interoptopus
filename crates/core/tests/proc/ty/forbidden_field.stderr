error: Using the name 'new' can cause conflicts in generated code.
 --> tests/proc/ty/forbidden_field.rs:5:5
  |
5 |     new: u8,
  |     ^^^
